CHANGELOG.md

```diff
@@ -186,6 +186,15 @@ Ref: https://keepachangelog.com/en/1.0.0/
     * The signature of `NewTxConfigWithTextual` has been deprecated and its signature changed to accept a `SignModeOptions`.
 * (x/genutil) [#15999](https://github.com/cosmos/cosmos-sdk/pull/15999) Genutil now takes the `GenesisTxHanlder` interface instead of deliverTx. The interface is implemented on baseapp
 * (types/math) [#16040](https://github.com/cosmos/cosmos-sdk/pull/16040) Remove unused aliases in math.go
+* [#15284](https://github.com/cosmos/cosmos-sdk/pull/15284)
+  * `sdk.Msg.GetSigners` was deprecated and is no longer supported. Use the `cosmos.msg.v1.signer` protobuf annotation instead.
```

Comment:
we should mention the extension interface as an alternative.

Reply:
would be good to understand how this works with an extension interface, i cant see anything in the pr about this but may have missed it

Reply:
for evm chains its needed, but unsure who else will need it in the future

Reply:
yea we can do follow up, but need to do it before release

Reply:
#16112